### DISCUSSION TITLE
Bug(scope):An unexpected error occurred: "https://registry.npmjs.org/@ngx-transloco%2fcore: Not found"

### DISCUSSION THREAD

**Discussion by lelelethu** [ngneat/transloco#672] [General]
When i install yarn add @ngneat/transloco, so "An unexpected error occurred: "https://registry.npmjs.org/@ngx-transloco%2fcore: Not found"".
Is there anyone like me?

**Comment by shaharkazaz** [comment]
I just ran `yarn add @ngneat/transloco` and it worked as expected on a fresh projet.

**Comment by lelelethu** [comment]
Please tell me where you are region?

Vào 21:27, Th 6, 28 thg 7, 2023 Shahar Kazaz ***@***.***> đã
viết:

**Comment by lelelethu** [comment]
@shaharkazaz You use what project and what region you live? I use angular and try with vietnam, america, germeni, chile but not working?

**Comment by lelelethu** [comment]
I found exactly @ngx-transloco/core not found,.
I try with fresh project so yarn add @ngneat/transloco work fine.
But my project so don't work

**Reply by shaharkazaz** [reply]
I'm not sure I follow, maybe you have some registry configuration that's making this issue? 
Without any details and the fact that it's working on a fresh project there isn't much to do.

**Comment by lelelethu** [comment]
i remove @ngx-transloco/core and run yarn again so it worrk.
